### PR TITLE
NMS-14150: Update XSD URL

### PR DESCRIPTION
**Pull request by emarsh1995**
- Updated and formatted operation/provisioning/introduction.adoc
- Updated XSD URL

### All Contributors

* [x] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [x] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* **If this code is likely to affect the UI, did you name your branch with `-smoke` in it to trigger smoke tests?**
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here, or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready. Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): https://issues.opennms.org/browse/NMS-14150

### CODE REVIEW

File: docs/modules/operation/pages/provisioning/introduction.adoc [hunk 1]
```diff
@@ -2,7 +2,7 @@
 [[ga-provisioning-introduction]]
 = Provisioning
 
-Provisioning is a mechanism to import node and service definitions either from an external source such as DNS or HTTP or via the {page-component-title} web UI.
+Provisioning is a mechanism that imports node and service definitions either from an external source such as DNS or HTTP or via the {page-component-title} web UI.
```
Comment (mmahacek) [inline docs/modules/operation/pages/provisioning/introduction.adoc:5]:
```suggestion
Provisioning is a process that imports node and service definitions either from an external source such as DNS or HTTP or via the {page-component-title} web UI.
```

File: docs/modules/operation/pages/provisioning/introduction.adoc [hunk 2]
```diff
@@ -26,45 +26,46 @@ Regardless of the method, provisioning is an iterative process: you will need to
 
 {page-component-title} uses an ICMP ping sweep to find IP addresses on the network and provision node entities.
 Use <<provisioning/auto-discovery.adoc#ga-provisioning-auto-discovery-detectors, auto discovery with detectors>> in addition to the ICMP IP address ping to specify services that you need to detect.
-xref:reference:provisioning/handlers/introduction.adoc#import-handlers[Import handlers] let you to further control provisioning.
+Use xref:reference:provisioning/handlers/introduction.adoc#import-handlers[import handlers] to further control provisioning.
```
Comment (mmahacek) [inline docs/modules/operation/pages/provisioning/introduction.adoc:29]: Import handlers are for getting nodes from different types of sources, typically those that aren't already xml formatted. Policies do more to control what is imported.

**Review by mmahacek — COMMENTED** [review]
First pass

File: docs/modules/operation/pages/provisioning/introduction.adoc (hunk 2 above)
Comment (emarsh1995) [reply docs/modules/operation/pages/provisioning/introduction.adoc:29]: Hmmm... okay. Does this look like a better description, then?

```suggestion
Use xref:reference:provisioning/handlers/introduction.adoc#import-handlers[import handlers] to provision nodes from other sources, including sources that may not be in XML format.
```

Comment (mmahacek) [reply docs/modules/operation/pages/provisioning/introduction.adoc:29]: I'm good with that wording.

File: docs/modules/operation/pages/provisioning/introduction.adoc [hunk 3]
```diff
@@ -13,11 +13,11 @@ It also includes an adapter API for interfacing with other management systems su
 
 Provisiond receives requests to add managed entities (nodes, IP interfaces, SNMP interfaces, services) via three basic mechanisms:
 
-* xref:provisioning/auto-discovery.adoc[automatic discovery] (typically via the Discovery daemon)
-* xref:provisioning/directed-discovery.adoc[directed discovery] using an import requisition (typically via the Provisioning UI)
-* xref:development:rest/nodes.adoc[node asset import] through the RestAPI or the https://docs.opennms.com/pris/1.2.0/index.html[provisioning integration server] (PRIS)
+* xref:provisioning/auto-discovery.adoc[Automatic discovery] (typically via the Discovery daemon).
+* xref:provisioning/directed-discovery.adoc[Directed discovery] using an import requisition (typically via the Provisioning UI).
+* xref:development:rest/nodes.adoc[Node asset import] through the RestAPI or the https://docs.opennms.com/pris/1.2.0/index.html[provisioning integration server] (PRIS).
```
Comment (Bonrob2) [inline docs/modules/operation/pages/provisioning/introduction.adoc:18]:
```suggestion
* xref:development:rest/nodes.adoc[Node asset import] through the REST API or the https://docs.opennms.com/pris/1.2.0/index.html[provisioning integration server] (PRIS).
```

File: docs/modules/operation/pages/provisioning/introduction.adoc [hunk 4]
```diff
@@ -26,45 +26,46 @@ Regardless of the method, provisioning is an iterative process: you will need to
...
+Directed discovery lets you specify what you want to provision based on an existing data source such as an in-house inventory, a standalone provisioning system, or a set of element management systems.
 Using an import requisition, this mechanism directs {page-component-title} to add, update, or delete a node entity exactly as defined by the external source.
 No discovery process is used for finding more interfaces or services.
 
 IMPORTANT: An import requisition is an XML definition of node entities to be provisioned from an external source into {page-component-title}.
-See the https://raw.githubusercontent.com/OpenNMS/opennms/develop/opennms-provision/opennms-provision-persistence/src/main/resources/xsds/model-import.xsd[requisition schema] (XSD) for more information.
+See the https://xmlns.opennms.org/xsd/config/model-import[requisition schema] (XSD) for more information.
 
-=== Understanding the Process
+=== Understanding the process
```
Comment (Bonrob2) [inline docs/modules/operation/pages/provisioning/introduction.adoc:51]:
```suggestion
=== The process
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Looks good. Minor changes!

File: docs/modules/operation/pages/provisioning/introduction.adoc [hunk 5]
```diff
@@ -2,7 +2,7 @@
 [[ga-provisioning-introduction]]
 = Provisioning
 
-Provisioning is a mechanism that imports node and service definitions either from an external source such as DNS or HTTP or via the {page-component-title} web UI.
+Provisioning is a process that imports node and service definitions either from an external source such as DNS or HTTP or via the {page-component-title} web UI.
```
Comment (mmahacek) [inline docs/modules/operation/pages/provisioning/introduction.adoc:5]:
```suggestion
Provisioning is a process that imports node and service definitions either from an external source, such as DNS or HTTP, or via the {page-component-title} web UI.
```

File: docs/modules/operation/pages/provisioning/introduction.adoc [hunk 6]
```diff
@@ -26,7 +26,7 @@ Regardless of the method, provisioning is an iterative process: you will need to
 
 {page-component-title} uses an ICMP ping sweep to find IP addresses on the network and provision node entities.
 Use <<provisioning/auto-discovery.adoc#ga-provisioning-auto-discovery-detectors, auto discovery with detectors>> in addition to the ICMP IP address ping to specify services that you need to detect.
-Use xref:reference:provisioning/handlers/introduction.adoc#import-handlers[import handlers] to further control provisioning.
+Use xref:reference:provisioning/handlers/introduction.adoc#import-handlers[import handlers] to provision nodes from other sources, including sources that may not be in XML format.
```
Comment (mmahacek) [inline docs/modules/operation/pages/provisioning/introduction.adoc:29]: Should be a separate paragraph, as it's a different kind of "automatic discovery"
```suggestion

Use xref:reference:provisioning/handlers/introduction.adoc#import-handlers[import handlers] to provision nodes from other data sources such as DNS zone records and VMware.
```

**Review by Bonrob2 — APPROVED** [review]
LGTM.

**Review by mmahacek — APPROVED** [review]
LGTM

**Pull request merged**